config/__tests__/__snapshots__/config.prod.test.js.snap

```diff
@@ -0,0 +1,519 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`prod configuration should load correct configuration: prod 1`] = `
+Config {
+  "authsome": Object {
+    "isPublic": true,
+    "mode": "/home/peter/dev/elife-xpub/config/non-serializable/authsome.js",
```

Comment:
besides helping to commit credentials, the snapshots seems also dependent on the machine where they were generated; so as for the other PR, I'd remove them and only check correct loading, not values

Reply:
@giorgiosironi ... check it out now